jdi-light-mobile-examples/src/main/java/io/github/com/pages/Header.java

```diff
@@ -0,0 +1,29 @@
+package io.github.com.pages;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import io.github.com.sections.LoginForm;
+import org.openqa.selenium.WebElement;
+
+public class Header extends Section {
+	@Css("form") public static LoginForm loginForm;
+	@Css(".fa-sign-out") public static WebElement logout;
+	@Css("#epam_logo") public static WebElement epamLogo;
+	public static WebElement userIcon;
+	@XPath("//*[@ui='label']") public static UIElement userName;
+
+	@Css(".search-field input") static UIElement searchInput;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)